meetings/meetings/settings/defaults.py

```diff
@@ -185,6 +195,11 @@
 
 OSF_API_URL = ('https://staging-api.osf.io').rstrip('/') + '/'
 OSF_ACCOUNTS_URL = ('https://staging-accounts.osf.io').rstrip('/') + '/'
+OSF_FILES_URL = os.environ.get(
```

Comment:
My bad. I just copied and pasted what the other files had for building urls didn't really pay attention to this. Thanks for the catch!